app/scripts/modules/titus/src/pipeline/stages/runJob/RunJobExecutionDetails.tsx

```diff
@@ -6,21 +6,24 @@ import {
   AccountTag,
   ExecutionDetailsSection,
   IExecutionDetailsSectionProps,
+  IExecutionDetailsComponentProps,
   StageFailureMessage,
 } from '@spinnaker/core';
 
 export interface IRunJobExecutionDetailsState {
   titusUiEndpoint?: string;
 }
 
+export interface IRunJobExecutionDetailsProps extends IExecutionDetailsComponentProps, IExecutionDetailsSectionProps {}
```

Comment:
not sure why this was needed but the TS compiler was not enjoying the IExecutionDetailsSectionProps